models/issues/review.go

```diff
@@ -159,6 +159,14 @@ func (r *Review) LoadReviewer(ctx context.Context) (err error) {
 		return err
 	}
 	r.Reviewer, err = user_model.GetPossibleUserByID(ctx, r.ReviewerID)
```

Comment:
Optimally, these changes should be part of `GetPossibleUserByID` (except for setting `r.ReviewerID`).
However, I don't know if there are any places whose behavior would change because of this.

Reply:
Agree, but it's still mixed logic like the change for the `comment.Assignee` which does not use `GetPossibleUserByID`.